### PR TITLE
Vendors: Create vendor from CollectivePicker mini-form

### PR DESCRIPTION
**Pull request by kewitz**
Requires https://github.com/opencollective/opencollective-api/pull/9494

### CODE REVIEW

File: components/vendors/VendorForm.tsx [hunk 1]
```diff
@@ -146,9 +147,9 @@ const VendorForm = ({ vendor, host, onSuccess, onCancel, isModal }: VendorFormPr
     <div>
       <div className="mb-3 flex justify-between text-xl font-bold">
         {vendor ? (
-          <FormattedMessage defaultMessage="Edit Vendor" />
+          <FormattedMessage id="vendor.edit" defaultMessage="Edit Vendor" />
         ) : (
-          <FormattedMessage defaultMessage="Create Vendor" />
+          <FormattedMessage id="vendor.create" defaultMessage="Create Vendor" />
```
Comment (Betree) [inline components/vendors/VendorForm.tsx:152]: Adding `id`s will force translators to re-translate the strings. Is it necessary here?

**Review by Betree — COMMENTED** [review]
The `VendorForm` is much clearer with the optional labels

File: components/vendors/VendorForm.tsx (hunk 1 above)
Comment (kewitz) [reply components/vendors/VendorForm.tsx:152]: Good point, I'll just remove it elsewhere. I thought this was too new to have been translated, but I was wrong.

Comment (Betree) [reply components/vendors/VendorForm.tsx]: These contributors are fast :-)

**Pull request merged**